demo/js/sentryMiddleware.js

```diff
@@ -0,0 +1,48 @@
+const identity = x => x;
+const getUndefined = () => {};
```

Comment:
Hi @captbaritone, according to their type of breadcrumbs data, this should return `undefined` instead of `void`.

Reply:
My understanding was that`void` is the type for `undefined`. This function implicitly returns undefined. I supposed it could be better to make it explicitly return undefined, but the actual behavior would be the same.

Reply:
The behavior should be the same, the type, specifically in the ts is different because they distinguish between no return value and undefined. But it's irrelevant in this case, my bad, sorry, the ts forced me to think about this differently.

Reply:
Interesting. TIL that in TypeScript you can have a return type of `undefined` which is incompatible with an implicit return of `undefined`.

Thanks!

<img width="236" alt="Screen Shot 2019-12-04 at 6 14 59 AM" src="https://user-images.githubusercontent.com/162735/70149856-ccaed180-165d-11ea-9505-e44243d79544.png">